packages/atomic/src/components/atomic-layout/__snapshots__/search-layout.spec.ts.snap

```diff
@@ -2,15 +2,17 @@
 
 exports[`Search Layout #buildSearchLayout when the atomic-search-layout component contains a main & facets sections
     should match snapshot 1`] = `
-"atomic-search-layout#abc { display: grid }@media only screen and (min-width: 1024px) {
+"atomic-search-layout#abc { display: grid }
```

Comment:
From a maintainer's perpective, I found the snapshot to be much more useful than any visual comparison.